.circleci/config.yml

```diff
@@ -279,19 +279,11 @@ jobs:
       - checkout
       - install_js:
           browsers: true
-      - run:
-          name: Run performance tests
-          command: xvfb-run yarn test:performance
```

Comment:
These scripts could be removed as well:
https://github.com/mui/mui-x/blob/6561be7ac4c5bc83f71d62cc12b2cc4e0e81b70e/package.json#L51-L55